test/e2e/placement_selecting_resources_test.go

```diff
@@ -674,6 +674,71 @@ var _ = Describe("validating CRP when selecting a reserved resource", Ordered, f
 	})
 })
 
+var _ = Describe("When creating a pickN ClusterResourcePlacement with duplicated resources", Ordered, func() {
+	crpName := fmt.Sprintf(crpNameTemplate, GinkgoParallelProcess())
+	var existingNS corev1.Namespace
+	BeforeAll(func() {
+		By("creating work resources on hub cluster")
+		createWorkResources()
+		existingNS = appNamespace()
+		By("Create a crp that selects the same resource twice")
+		crp := &placementv1beta1.ClusterResourcePlacement{
+			ObjectMeta: metav1.ObjectMeta{
+				Name: crpName,
+				// Add a custom finalizer; this would allow us to better observe
+				// the behavior of the controllers.
+				Finalizers: []string{customDeletionBlockerFinalizer},
+			},
+			Spec: placementv1beta1.ClusterResourcePlacementSpec{
+				ResourceSelectors: []placementv1beta1.ClusterResourceSelector{
+					{
```

Comment:
should we block this case via the validation webhook?

Reply:
I think so, https://github.com/kubefleet-dev/kubefleet/issues/13